pdfmaker/kassia.py

```diff
@@ -44,10 +45,15 @@ def setDefaults(self):
         self.pageAttrib['line_height'] = 72
         self.pageAttrib['line_width'] = self.pageAttrib['paper_size'][0] - (self.pageAttrib['left_margin'] + self.pageAttrib['right_margin'])
```

Comment:
If someone wanted to create Zoe style books, the default margins and spacing would probably be a little smaller. I'd say it's pretty low priority, tho. Yes, Reportlab has a bunch of preset sizes (A4, legal, letter) and lets you customize it, but we'd only need defaults for a couple of them with a catch-all (probably 1" margins).